Packs/AWS/Integrations/AWS/AWS.py

```diff
@@ -749,6 +749,95 @@ def put_bucket_ownership_controls_command(client: BotoClient, args: Dict[str, An
         except Exception as e:
             raise DemistoException(f"Error: {str(e)}")
 
+    @staticmethod
+    def get_bucket_website_command(client: BotoClient, args: Dict[str, Any]) -> CommandResults:
+        """
+        Retrieves the website configuration for a specified Amazon S3 bucket.
+        The function calls the AWS S3 'get_bucket_website' API to check if the bucket
+        is configured for static website hosting and, if so, what its configuration is.
+
+        Args:
+            client (BotoClient): The initialized Boto3 S3 client.
+            args: A dictionary containing arguments, expected to include 'bucket'
+                  (the name of the S3 bucket).
+
+        Returns:
+            CommandResults: An object containing the raw website configuration as outputs,
+                            and a md table summarizing the configuration details
+                            (IndexDocument, ErrorDocument, RedirectAllRequestsTo, RoutingRules).
+        """
+        kwargs = {"Bucket": args.get("bucket")}
+        try:
+            response = client.get_bucket_website(**kwargs)
+            if response["ResponseMetadata"]["HTTPStatusCode"] in [HTTPStatus.OK, HTTPStatus.NO_CONTENT]:
+                response["WebsiteConfiguration"] = {
+                    "ErrorDocument": response.get("ErrorDocument"),
+                    "IndexDocument": response.get("IndexDocument"),
+                    "RedirectAllRequestsTo": response.get("RedirectAllRequestsTo"),
+                    "RoutingRules": response.get("RoutingRules")
+                }
+                readable_output = tableToMarkdown(
+                    name="Bucket Website Configuration",
+                    t=response.get("WebsiteConfiguration", {}),
+                    removeNull=True,
+                    headers=["ErrorDocument", "IndexDocument", "RedirectAllRequestsTo", "RoutingRules"],
+                    headerTransform=pascalToSpace,
+                )
+                return CommandResults(readable_output=readable_output,
+                                      outputs_prefix="AWS.S3-Buckets.BucketWebsite",
+                                      outputs=response.get("WebsiteConfiguration", {}),
+                                      raw_response=response.get("WebsiteConfiguration", {}))
+            raise DemistoException(
+                f"AWS S3 API call to get_bucket_website failed or returned an unexpected status code. "
+                f"Received HTTP Status Code: {response['ResponseMetadata']['HTTPStatusCode']}"
+            )
```

Comment:
```suggestion
            raise DemistoException(
                f"AWS S3 API call to get_bucket_website failed or returned an unexpected status code. "
                f"Received HTTP Status Code: {response['ResponseMetadata']['HTTPStatusCode']}"
            )
```
Instead of raising an exception, please use the AWSErrorHandler.handle_response_error().